daemon/image_delete.go

```diff
@@ -124,9 +116,8 @@ func (daemon *Daemon) DeleteImage(eng *engine.Engine, name string, list *[]types
 				Deleted: img.ID,
 			})
 			daemon.EventsService.Log("delete", img.ID, "")
-			eng.Job("log", "delete", img.ID, "").Run()
```

Comment:
Yes, it is dead code. You can remove it if you want. Or can create issue for beginner contributors.